manifests/s/Sass/Dart-Sass/1.83.4/Sass.Dart-Sass.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Sass.Dart-Sass
+PackageVersion: 1.83.4
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: dart-sass\src\dart.exe

```

```suggestion
- RelativeFilePath: dart-sass/sass.bat
```

We're NOT asking for the Dart executable, but the Saas compiler.